url-shortner.js

```diff
@@ -0,0 +1,52 @@
+/*
+1. http://localhost:3000/ -> we get a form
+2. we submit the form
+3. app generates an random id like 234234 => function storeShortUrl
+4. return the the front page where we see the new short url in the table below
+5. when we visit http://localhost:3000/234234 it redirects to the full url.
+*/
+
+const express = require("express");
+const app = express();
+const sql = require("./sqlFile");
```

Comment:
If you rename sqlFile.js to db.js, it makes senses to call this variable `db` instead of `sql`.